tests/test_game.py

```diff
@@ -10,7 +10,7 @@ def __init__(self):
     def is_full(self):
         return False
 
-    def occupy_space(self, space_index, symbol):
+    def place_marker(self, space_index, symbol):
```

Comment:
I like this change. I wonder if you could update function parameter names and body to reflect and keep coherent?